src/translations/commons.ts

```diff
@@ -20,9 +21,11 @@ export type TranslateFunction = TFunc<typeof Language>;
 export function translation(
   norwegian: string,
   english: string,
+  nynorsk: string,
 ): TranslatedString {
   return {
     [Language.Norwegian]: norwegian,
     [Language.English]: english,
+    [Language.Nynorsk]: nynorsk === '§' ? norwegian : nynorsk,
```

Comment:
Do you think the approach with `'§'` strings as a sign of approval for situations where nynorsk and bokmål strings are equal is maintainable, @mikaelbr? 

The idea is to avoid having to write duplicate strings whenever bokmål and nynorsk is the same.